tests/core/tests/test_admin_integration.py

```diff
@@ -680,7 +680,10 @@ def test_export(self):
             "file_format": "0",
         }
         date_str = datetime.now().strftime("%Y-%m-%d")
-        response = self.client.post("/admin/core/book/export/", data)
+        with self.assertNumQueries(
```

Comment:
I would prefer a more robust check to prove that `show_full_result_count` and `get_paginator` were not called - perhaps they could be mocked?

I think that counting the number of queries is prone to failing in future when Django is updated.